src/k8s/api/v1/types.go

```diff
@@ -77,8 +77,9 @@ func (c ClusterStatus) datastoreToString() string {
 		result.WriteString(fmt.Sprintf("  type: %s\n", c.Datastore.Type))
 		// Datastore URL for external only
 		if c.Datastore.Type == "external" {
-			if c.Datastore.ExternalURL != "" {
-				result.WriteString(fmt.Sprintf("  url: %s\n", c.Datastore.ExternalURL))
+			result.WriteString(fmt.Sprintln("  servers:"))
+			for _, serverURL := range c.Datastore.ExternalServers {
+				result.WriteString(fmt.Sprintf("    - %s\n", serverURL))
```

Comment:
Here -- in the "plain-text rendering" -- it looks like `servers`.  that's great.  Would love the same from the API